store/mockstore/unistore/tikv/mvcc_test.go

```diff
@@ -1625,3 +1625,55 @@ func TestAccessCommittedLocks(t *testing.T) {
 		}
 	}
 }
+
+func TestTiKVRCRead(t *testing.T) {
+	store, close := NewTestStore("basic_optimistic_db", "basic_optimistic_log", t)
+	defer close()
+
+	k1 := []byte("t1")
+	k2, v2 := []byte("t2"), []byte("v2")
+	k3, v3 := []byte("t3"), []byte("v3")
+	k4, v4 := []byte("t4"), []byte("v4")
+	MustLoad(10, 20, store, "t1:v1", "t2:v2", "t3:v3")
+	// write to be read
+	MustPrewritePut(k1, k1, []byte("v11"), 30, store)
+	MustCommit(k1, 30, 40, store)
+	// lock to be ignored
+	MustPrewritePut(k2, k2, v2, 50, store)
+	MustPrewriteDelete(k3, k3, 60, store)
+	MustPrewritePut(k4, k4, v4, 70, store)
+
+	expected := map[string][]byte{string(k1): []byte("v11"), string(k2): v2, string(k3): v3, string(k4): nil}
+
+	reqCtx := store.newReqCtx()
+	reqCtx.rpcCtx.IsolationLevel = kvrpcpb.IsolationLevel_RC
+	// get
+	for k, v := range expected {
+		res, err := store.MvccStore.Get(reqCtx, []byte(k), math.MaxUint64)
```

Comment:
Using a timestamp which is not that special would be better, in case `MaxUint64` implies RC semantics.